_includes/docs/latest/setup-script-credentials-gke.md

```diff
@@ -2,6 +2,10 @@
 . ./helpers/setup.sh -p PROJECT_ID -o ORG_ID -k
 ```
 
-This will create a service account called `cloud-foundation-forseti-<suffix>`,
-assign it the proper roles, and download the service account credentials to
-`${PWD}/credentials.json`.
+This will create a service account called `cloud-foundation-forseti-<suffix>`
+and assign it the necessary roles. 
+
+Set and export the `GOOGLE_APPLICATION_CREDENTIALS` variable to the newly
+created credential.json as shown below.
+
+`export GOOGLE_APPLICATION_CREDENTIALS="/path/to/credentials.json"`
```

Comment:
Is this already covered in the setup.sh script? https://github.com/forseti-security/terraform-google-forseti/blob/1793f0b0cf75dccf7b2d56759b886a7b1c5dfd7d/helpers/setup.sh#L159

Reply:
Yes, removed it in the latest and other version docs.